lib/eco/treemanager.py

```diff
@@ -1737,44 +1798,74 @@ def export_aterms(self, path):
 
     def relex(self, node):
         if node is None:
-            return
+            return False
         if isinstance(node, BOS) or isinstance(node, EOS):
-            return
+            return False
         if isinstance(node.symbol, MagicTerminal):
-            return
+            return False
+        try:
+            return self.relex_node(node)
+        except LexingError:
+            #XXX show LexingError message somwhere in the UI
```

Comment:
For now they are, but I was planning to mark them somehow. My initial thought was to also not parse if there is a lexing error, but when we do the partial lexing stuff, then we need to reparse anyway.